src/diffusers/models/attention_processor.py

```diff
@@ -78,6 +78,7 @@ def __init__(
         self.upcast_softmax = upcast_softmax
         self.rescale_output_factor = rescale_output_factor
         self.residual_connection = residual_connection
+        self.dropout = dropout
```

Comment:
Added so can re-create the dropout module when converting back to new weight format